core/src/validator.rs

```diff
@@ -601,15 +601,15 @@ impl Validator {
             &config.snapshot_config.bank_snapshots_dir,
         );
 
-        info!("Cleaning orphaned account snapshot directories..");
+        info!("Cleaning orphaned account snapshot directories...");
         let mut timer = Measure::start("clean_orphaned_account_snapshot_dirs");
         clean_orphaned_account_snapshot_dirs(
             &config.snapshot_config.bank_snapshots_dir,
             &config.account_snapshot_paths,
         )
-        .map_err(|err| format!("Failed to clean orphaned account snapshot directories: {err:?}"))?;
+        .map_err(|err| format!("failed to clean orphaned account snapshot directories: {err}"))?;
         timer.stop();
-        info!("Cleaning orphaned account snapshot directories done. {timer}");
+        info!("Cleaning orphaned account snapshot directories... Done, and{timer}");
```

Comment:
nit: The old string, `... snapshot directories done. {timer}` is consistent with several other logs in this same file; you can grep `done` to see those